packages/component-library/src/components/plmg-button/plmg-button.scss

```diff
@@ -363,26 +372,94 @@
     }
   }
 
-  // Control: Size
+  // Control: Size & icons
   &.small {
     padding: tokens.$plmg-spacing-x0-5 tokens.$plmg-spacing-x1;
     font-size: tokens.$plmg-font-size-x0-875;
     line-height: tokens.$plmg-font-lineheight-loose;
+
+    &.icon-button {
+      padding: tokens.$plmg-spacing-x1;
+    }
+
+    .icon-left {
+      margin-right: tokens.$plmg-spacing-x0-5;
+    }
+    .icon-right {
+      margin-left: tokens.$plmg-spacing-x0-5;
+    }
+    .icon-center {
+      font-size: centerIconFontSize(
+        tokens.$plmg-font-size-x0-875,
+        tokens.$plmg-font-lineheight-loose
+      );
+    }
   }
   &.medium {
     padding: tokens.$plmg-spacing-x0-75 tokens.$plmg-spacing-x1-5;
     font-size: tokens.$plmg-font-size-x1-125;
     line-height: tokens.$plmg-font-lineheight-loose;
+
+    &.icon-button {
+      padding: tokens.$plmg-spacing-x0-75;
+    }
+
+    .icon-left {
+      margin-right: tokens.$plmg-spacing-x0-75;
+    }
+    .icon-right {
+      margin-left: tokens.$plmg-spacing-x0-75;
+    }
+    .icon-center {
+      font-size: centerIconFontSize(
+        tokens.$plmg-font-size-x1-125,
+        tokens.$plmg-font-lineheight-loose
```

Comment:
Both medium and large icon center buttons are slightly off in size. 
Is there a pixel margin of error or is the design system intended to be pixel perfect to the design.

Reply:
The goal is for them to have the exact same size as a Normal Button of the same size
![image](https://user-images.githubusercontent.com/55230735/160587746-baa435d8-dfdf-4d2e-98f1-dedfccfb81c2.png)

Which they now do :-)